trio/_path.py

```diff
@@ -182,6 +182,73 @@ async def open(self, *args, **kwargs):
         value = await trio.to_thread.run_sync(func)
         return trio.wrap_file(value)
 
+    if TYPE_CHECKING:
+        # the dunders listed in _forward_magic that aren't seen otherwise
+        __bytes__ = pathlib.Path.__bytes__
+        __truediv__ = pathlib.Path.__truediv__
+        __rtruediv__ = pathlib.Path.__rtruediv__
+
+        # Note: these are not parsed properly by jedi.
```

Comment:
I think this comment is outdated?

Reply:
I at least don't remember why I wrote it, so removing it for now.